roles/edpm_libvirt/meta/argument_specs.yml

```diff
@@ -0,0 +1,7 @@
+---
+argument_specs:
+  # ./roles/edpm_libvirt/tasks/main.yml entry point
+  main:
+    short_description: The main entry point for the edpm_libvirt role.
+    description: Multiple lines description
+    options: {}
```

Comment:
Role arg should be documented here. If they are all optional we can mark them as such.

Reply:
yep as noted prevsiouly this role is part of a separate PR please review it there #215 instead not here